easybuild/easyconfigs/p/Python/Python-3.9.5-GCCcore-10.3.0-bare.eb

```diff
@@ -0,0 +1,34 @@
+name = 'Python'
+version = '3.9.5'
+versionsuffix = '-bare'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['e0fbd5b6e1ee242524430dee3c91baf4cbbaba4a72dd1674b90fda87b713c7ab']
+
+builddependencies = [
+    ('UnZip', '6.0'),
+]
+
+dependencies = [
+    ('binutils', '2.36.1'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.1'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.35.4'),
+    ('XZ', '5.2.5'),
+    ('libffi', '3.3'),
```

Comment:
the `# required for cryptography` comment in the full Python easyconfigs is now misleading, libffi is now also required to build Python itself - Python used to bundle libffi, but not anymore (since 3.7?)

Reply:
Then we should tweak the comment there to reflect that (best in a separate PR)?

Reply:
confirmed the change was on 3.7.0, will do

Reply:
https://github.com/easybuilders/easybuild-easyconfigs/pull/13041